CHANGELOG.md

```diff
@@ -1,108 +1,222 @@
 Changelog
 =========
 
+Version 0.11 (June 20, 2022)
+----------------------------
+
+Highlights of this release:
+
+- GeoParquet support updated to implement
+  [v0.4.0](https://github.com/opengeospatial/geoparquet/releases/tag/v0.4.0) of the
+  OpenGeospatial/GeoParquet specification (#2441).Backwards compatibility with v0.1.0 of
+  the metadata spec (implemented in the previous releases of GeoPandas) is guaranteed,
+  reading and writing Parquet and Feather files will no longer produce a ``UserWarning``
+  (#2327).
+
+New features and improvements:
+
+- Improved handling of GeoDataFrame when the active geometry column is
+  lost from the GeoDataFrame. Previously, square bracket indexing ``gdf[[...]]`` returned
+  a GeoDataFrame when the active geometry column was retained and a DataFrame was
+  returned otherwise. Other pandas indexing methods (``loc``, ``iloc``, etc) did not follow
+  the same rules. The new behaviour for all indexing/ reshaping is (#2329, #2060) is as
+  follows:
+  - If operations produce a ``DataFrame`` containing the active geometry column, a
+    GeoDataFrame is returned
+  - If operations produce a ``DataFrame`` containing ``GeometryDtype`` columns, but not the
+    active geometry column, a ``GeoDataFrame`` is returned, where the active geometry
+    column is set to ``None`` (set the new geometry column with ``set_geometry()``)
+  - If operations produce a ``DataFrame`` containing no ``GeometryDtype`` columns, a
+    ``DataFrame`` is returned (this can be upcast again by calling ``set_geometry()`` or the
+    ``GeoDataFrame`` constructor)
+  - If operations produce a ``Series`` of ``GeometryDtype``, a ``GeoSeries`` is returned,
+    otherwise ``Series`` is returned. Error messages for having an invalid geometry column
+    have been improved, indicating the name of the last valid active geometry column set
+    and whether other geometry columns can be promoted to the active geometry column
+    (#2329).
```

Comment:
```suggestion
    otherwise ``Series`` is returned.
  - Error messages for having an invalid geometry column
    have been improved, indicating the name of the last valid active geometry column set
    and whether other geometry columns can be promoted to the active geometry column
    (#2329).
```